js/menyConfig.js

```diff
@@ -7,7 +7,7 @@ const appDomain = `https://app${finnMiljoStreng()}.adeo.no`;
 const tjenesterDomain = `https://tjenester${finnMiljoStreng()}.nav.no`;
 const naisDomain = `.nais.${finnNaisMiljoStreng()}`;
 const frontendLoggerApiEvent = '/frontendlogger/api/event';
-
+const arbeidstreningDomain = `https:/arbeidsgiver.nais.${NAIS_PROD_ARBEIDSGIVER}`;
```

Comment:
Denne vil vel alltid peke mot produksjons-miljø? Burde nok heller gjøre som de andre urlene her, slik at testing i QXXX miljøer er enklere.

Reply:
Høres det greit ut hvis jeg bruker samme "finnNaidMiljoStreng" fun siden vi har samme nais domen ? eller er det bedre at vi lager egen "finnNaisMiljoStren" for arbedisgiver tjenester?